src/assignment/views/AssignmentEdit.tsx

```diff
@@ -132,6 +135,21 @@ const AssignmentEdit: FunctionComponent<DefaultSecureRouteProps<{ id: string }>>
 
 	// Events
 
+	const handleOnSave = async () => {
+		if (!user.profile?.id || !original) {
+			return;
+		}
+
+		const responses = await AssignmentService.getAssignmentResponses(
+			user.profile?.id,
+			original.id
+		);
+		if (responses.length > 0) {
+			setIsConfirmSaveActionModalOpen(true);
+			return;
+		}
```

Comment:
shouldn't there also be an else here?
where you save the assignment right away?

Reply:
indeed, fiexed